polkadot/statement-table/src/generic.rs

```diff
@@ -408,33 +396,7 @@ impl<Ctx: Context> Table<Ctx> {
 				// if digest is different, fetch candidate and
 				// note misbehavior.
 				let existing = occ.get_mut();
-
-				if !self.config.allow_multiple_seconded && existing.proposals.len() == 1 {
-					let (old_digest, old_sig) = &existing.proposals[0];
-
-					if old_digest != &digest {
-						const EXISTENCE_PROOF: &str =
-							"when proposal first received from authority, candidate \
-							votes entry is created. proposal here is `Some`, therefore \
-							candidate votes entry exists; qed";
-
-						let old_candidate = self
-							.candidate_votes
-							.get(old_digest)
-							.expect(EXISTENCE_PROOF)
-							.candidate
-							.clone();
-
-						return Err(Misbehavior::MultipleCandidates(MultipleCandidates {
```

Comment:
The `MultipleCandidates` struct can be removed along with `Misbehavior::MultipleCandidates`

Reply:
Done